bin/attach-dev

```diff
@@ -20,4 +20,4 @@ echo "Civiform starting locally at http://localhost:9000/loginForm"
 
 echo "Run bin/stop-dev to stop the server"
 
-docker attach civiform
+docker attach "${COMPOSE_PROJECT_NAME}-civiform-1"
```

Comment:
i only see this getting set for the tests is it supposed to get set somewhere for just running these commands too?

Reply:
If you checkout civiform in a different folder, docker uses that folder as the prefix.  COMPOSE_PROJECT_NAME captures that and generates the container name accordingly.

Reply:
It's mostly just unfortunate that there's no docker-compose command for this